### ISSUE TITLE
Module 6: Ensemble of Models

### ISSUE THREAD

**Issue by anu-coder** [anu-coder/scikit-learn-ML#6]
- [x] Intuitions on Ensemble models
- [x] Ensemble models uing bootstrapping
        - M6: Notebook 1: Bagging 
        - M6: Notebook 2: Random Forest
- [x] Ensemble based on boosting
        - M6: Notebook 3: AdaBoost
        - M6: Notebook 4: Gradient Boosting Decision Tree (GBDT)
        - M6: Notebook 5: Speeding up gradient boosting
- [x] Hyperparameter tuning ensemble models
        - M6: Notebook 6: Hyperparameter tuning
- [x] Quiz and take away
